x-pack/plugins/security_solution/cypress/e2e/detection_alerts/investigate_in_timeline.cy.ts

```diff
@@ -62,9 +68,39 @@ describe('Alerts timeline', () => {
   });
 
   it('Add an empty property to default timeline', () => {
+    // add condition to make sure the field is empty
```

Comment:
Why is this needed now?

Reply:
There are lot of records and we want to make sure that first row has empty value for `file.name` .  Safest method to ensure that is to add a filter. 

Now this situation arises sometimes when this test runs after another test which hasn't unloaded their own data. It looks like `cleankibana` does not clear the data uploaded by `esarchiver`